### PR TITLE
add gemma@1.1.0 w/ npm auto-update

### LINKED ISSUE #8990: [Request] Add gemma
**Library name:** gemma
**Git repository url:** https://github.com/colepeters/gemma

### PR DESCRIPTION
**Pull request by nolsherry**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/colepeters/gemma
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/gemma
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 86
   - Count of watchers: 12
   - Count of forks: 5
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 29
   - Downloads in the last month: 141

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8990, cc @colepeters

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@nolsherry I think you can just add `public/gemma.min.css` because it's a compiled and minified file.

File: ajax/libs/gemma/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "gemma",
+  "filename": "gemma.min.css",
+  "version": "1.1.0",
+  "description": "A lightweight CSS library",
+  "style": "public/gemma.min.css",
```
Comment (pvnr0082t) [inline ajax/libs/gemma/package.json]: Please remove `style` field here. Thanks.

**Comment by nolsherry** [comment]
@pvnr0082t, I have removed `style` field, and only added `public/gemma.min.css` in filemap

**Comment by pvnr0082t** [comment]
LGTM

**Comment by kennynaoh** [comment]
@nolsherry 
What about `v0.1.0`?
I found that there has no `gemma.min.css` file on both github and npm.
And in version `0.4.0`, the file `gemma.min.css`  didn't minified.
Please discuss with author.
Thanks!

**Comment by nolsherry** [comment]
@kennynaoh, the author have responsed the [issue](https://github.com/colepeters/gemma/issues/6), and the author don’t have any plans for maintaining or supporting the versions before v1.x.x. The author indicates these versions are effectively deprecated, and he would not recommend that anyone use them.
I thinck maybe we can help the author minified the `gemma.min.css.` in v0.4.1, and skip v0.1.0. Do you have any better idea?

cc @PeterDaveHello, cc @colepeters

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I am not sure about the file in `v0.1.0`
Because author doesn't have any plans about version before `1.x.x`.
What should do in version `v0.1.0` which has no main file?

And in `v0.4.0`, should we change the name to `.js` and minify it manually, or just leave it alone?
Thanks!

**Comment by PeterDaveHello** [comment]
We can ignore that two versions.

**Comment by nolsherry** [comment]
@PeterDaveHello, Should I need to send two commits? One is just for adding the latest version, and another is for adding odd versions with `v0.1.0`, `v0.4.0` added to`.gitignore` file.
Or should I just send a commit to add the latest version `v1.1.0` with `v0.1.0`, `v0.4.0` added to`.gitignore` file? Please give me some suggestions, thanks!

**Comment by PeterDaveHello** [comment]
One commit is enough, thanks.

**Comment by nolsherry** [comment]
@pvnr0082t, @kennynaoh, I have added the two versions(v0.1.0, v0.4.0) into `.gitignore` file, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**